### DISCUSSION TITLE
Are elliptic curve keys supported?

### DISCUSSION THREAD

**Discussion by Cedesc** [eclipse-milo/milo#1220] [General]
Hey, are EC keys supported? I have not found anything about this. But as far as I have seen, no appropriate security policy is supported, right?
Thank you!

**Comment by kevinherron** [comment]
No, the ECC security policies are not yet implemented. 

What server are you connecting to that supports them?

**Reply by Cedesc** [reply]
It was only theoretical for a self-hosted server. Thank you for the quick response!
